assets/javascripts/discourse/initializers/chat-keyboard-shortcuts.js

```diff
@@ -89,6 +98,11 @@ export default {
         (event) => modifyComposerSelection(event, "code"),
         { global: true }
       );
+      api.addKeyboardShortcut(
+        `${mod}+l`,
```

Comment:
I feel like I asked before.. do we register any chat shortcuts into a list?

Reply:
@nattsw you are right! I was looking at that help modal just this morning! Added a message for insert link now :)